wdl-analysis/src/document.rs

```diff
@@ -94,6 +96,16 @@ pub struct Struct {
 }
 
 impl Struct {
+    /// Gets the name of the struct.
```

Comment:
Can we order these accessor methods so they match the order of the members in the struct?